lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -1974,9 +1970,13 @@ void ThreadContext::ReleaseDebugManager()
         {
             this->recyclableData->returnedValueList = nullptr;
         }
-        this->debugManager->Close();
-        HeapDelete(this->debugManager);
-        this->debugManager = nullptr;
+
+        if (this->debugManager != nullptr)
```

Comment:
I do not understand the scenario here - this is the only place we delete the debugManager. which is when refcounter became 0. 
So at this point can it be null ? Or it is just conservative check (guarded by an assert above) ?